verification/curator-ui/pages/index.js

```diff
@@ -0,0 +1,35 @@
+import Head from 'next/head'
+
+export default function Home() {
+  return (
+    <div className="container">
```

Comment:
I don't speak React yet, but it looks a little weird that this is "className" instead of "class" ?

Reply:
Yea, that's React speak

Reply:
Just went through react tutorials today, I confirm
![image](https://user-images.githubusercontent.com/1255432/80211875-c6ce8280-8636-11ea-98a3-bce3e8dce1b8.png)